src/APIdata.js

```diff
@@ -0,0 +1,40 @@
+const ApiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+
+const MakeGame = () => {
+  fetch(
+    `${ApiUrl}games/`, {
+      method: 'POST',
+      body: JSON.stringify({
+        name: "franklin's new game",
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    },
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      localStorage.setItem('Game Index', data.result.split(' ')[3]);
+    });
+};
+
+const addScore = (gameIndex, name, score) => {
+  fetch(
+    `${ApiUrl}/games/${gameIndex}/scores`,
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        user: name,
+        score,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    },
+  )
+    .then((response) => response.json());
+};
```

Comment:
- [ ] Hi @franklinben23, from the requirements of this project, the submit button should hit the post API endpoint and post user entries. When I click on the submit button after entering the user name and score, I got a console error, and data is not reflected on the scoreboard. Please let's fix this issue.

Reply:
If you check the submit button hits the API when because its function its to call the getGames APImethod and it does print the user entries, so from that side its ok actually, furthermore, you did not specify the error and I don't have any on my machine.

Reply:
If you check the index.js file, it is doing exactly that, check the
`Form.addEventListener('submit', (e) => {
  e.preventDefault();
  if (Form.name.value === '' || Form.score.value === null) return;
  APIdata.addScore(gameIndex, Form.name.value, Form.score.value);
  render(gameIndex);
  clearElement(Scores);
});`

at line 46. furthermore, you did not specify the error and on my machine I don't have any.

Reply:
resolved 👍